docs/getting-started/create-an-app.md

```diff
@@ -108,7 +108,7 @@ example [here](https://www.python.org/downloads/) to install it.
 
 Install yarn on your system with `npm install --global yarn` or for more details
 refer to the
-[prerequisites](https://backstage.io/docs/getting-started/running-backstage-locally#prerequisites)
+[prerequisites](https://backstage.io/docs/getting-started/#prerequisites)
```

Comment:
```suggestion
[prerequisites](index.md#prerequisites).
```

Adds a period to the sentence, but we should rely on Docusaurus to set these URLs, not hardcode them. That way if the base URL changes or formatting in anyway, Docusaurrus should be able to properly regenerate them.

Reply:
Good catch, fixed! And yes there's a bit of everything in these docs - full URLs vs. pointing to the MD files